app/src/main/res/layout/profile_edit_fragment.xml

```diff
@@ -48,12 +48,15 @@
       <TextView
         android:id="@+id/profile_edit_name"
         style="@style/Heading2"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_marginStart="24dp"
+        android:layout_marginEnd="16dp"
         android:importantForAccessibility="no"
         android:text="@{viewModel.profile.name}"
+        android:textDirection="locale"
```

Comment:
I think instead of `textDirection`, `textAlignment` attribute would be a better choice.
Try following things:

1. Remove `android:textDirection="locale"` and recreate screenshots and see if the `after` result is same, if it works that great. If not go to (2.)
2. Use `android:textAlignment="viewStart"` instead of `android:textDirection="locale"` and recreate screenshots and see if the `after` result is same

Reply:
@rt4914 I tried both. Both of them has issues with RTL. 

![Screenshot_1641552090](https://user-images.githubusercontent.com/55173065/148532242-47806a04-b4c1-49d4-9e3e-9dd1b030d0d9.png)

Reply:
Just remove this attribute `textDirection` and instead use style `Heading2ViewStart`.

```
      <TextView
        android:id="@+id/profile_edit_name"
        style="@style/Heading2ViewStart"
        android:layout_width="0dp"
        android:layout_height="wrap_content"
        android:layout_marginStart="24dp"
        android:layout_marginEnd="16dp"
        android:importantForAccessibility="no"
        android:text="@{viewModel.profile.name}"
        app:layout_constraintBottom_toTopOf="@+id/profile_edit_created"
        app:layout_constraintEnd_toEndOf="parent"
        app:layout_constraintStart_toEndOf="@id/profile_edit_image"
        app:layout_constraintTop_toTopOf="@id/profile_edit_image"
        app:layout_constraintVertical_chainStyle="packed" />
```

This should give following result:

<img src="https://user-images.githubusercontent.com/9396084/149200874-ba23f617-bb0d-4a25-aca4-04b344364b2c.png" width="250" />

Reply:
Done !